edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/TwinConfigSource.cs

```diff
@@ -98,11 +101,32 @@ async Task<Option<EdgeHubConfig>> GetConfigInternal()
             {
                 Core.IMessage message = await this.twinManager.GetTwinAsync(this.id);
                 Twin twin = this.twinMessageConverter.FromMessage(message);
-                this.lastDesiredProperties = Option.Some(twin.Properties.Desired);
+                TwinCollection desiredProperties = twin.Properties.Desired;
```

Comment:
This section of code is very similar to the one at PatchDesiredProperties. Can we create a utility and reuse?